testing/ponysay/APKBUILD

```diff
@@ -0,0 +1,21 @@
+# Maintainer: Matthew T Hoare <matthew.t.hoare@gmail.com>
+# Contributor: Sven-Hendrik Haase <sh@lutzhaase.com>
+# Contributor: Erkin Batu Altunbaş <erkinbatu@gmail.com>
+
+pkgname=ponysay
+pkgver=3.0.2
+pkgrel=0
+pkgdesc="cowsay reimplemention for ponies"
+url="http://erkin.github.com/ponysay/"
+arch="noarch"
+license="GPL3"
+depends="python bash"
```

Comment:
texinfo seems to be a makedepends, I had to add it locally for the docs to be generated.